Tasks/JenkinsQueueJobV2/Tests/L0.ts

```diff
@@ -32,7 +32,7 @@ describe('JenkinsQueueJob L0 Suite', function () {
         try {
             tr.run();
 
-            assert(tr.stderr.indexOf('Input required: serverEndpoint') !== -1, 'should have written to stderr');
```

Comment:
After updating the vsts-task-lib I had to update the tests to reflect the removal of the writeError method.

Reply:
Use `tl.error` where it used to have `_writeError`.  That will give you the red error text in the console, though it won't fail the task.

![image](https://user-images.githubusercontent.com/33549821/51000736-f28a2f80-14fb-11e9-9ceb-704903cce2b1.png)